docs/guides/src/main/server/outgoinghttp.adoc

```diff
@@ -118,7 +118,7 @@ In this example, the following occurs:
 * The special value NO_PROXY for the proxy-uri is used, which means that no proxy is used for hosts matching the associated hostname pattern.
 * A catch-all pattern ends the proxy-mappings, providing a default proxy for all outgoing requests.
 
-== Outgoing HTTPS request truststore
+== Enabling TLS connections
```

Comment:
TLS connections are already enabled though? This is about adding additional certs that you trust to the store.

Reply:
On that note it would also be nice to have a non-Java way of trusting more certs ;)

Reply:
By the way there's this proposal from the community:
https://github.com/keycloak/keycloak-community/pull/345

Haven't looked at it yet though.